setup.py

```diff
@@ -45,7 +45,6 @@
     'packages': [
                     'pennylane',
                     'pennylane.ops',
-                    'pennylane.expval',

```

Comment:
add `pennylane.measure` here?

Reply:
This list is only for packages (directories), whereas `pennylane.measure` is a submodule